packages/php-db-import-export/src/Backend/Bigquery/ToFinalTable/SqlBuilder.php

```diff
@@ -146,6 +119,13 @@ public function getInsertAllIntoTargetTableCommand(
         BigqueryImportOptions $importOptions,
         string $timestamp,
     ): string {
+        $columnMap = SourceDestinationColumnMap::createForTables(
+            $sourceTableDefinition,
+            $destinationTableDefinition,
+            $importOptions->ignoreColumns(),
+            SourceDestinationColumnMap::MODE_MAP_BY_NAME,
+        );
```

Comment:
synchronizuju logiku stím jak je to teď ve SNFLK SourceDestinationColumnMap řeší ty ignore columns